### PR TITLE
VP-5619: Improve page titles

### PR DESCRIPTION
**Pull request by trueboroda**

### CODE REVIEW

File: snippets/meta/meta-info.liquid [hunk 1]
```diff
@@ -2,18 +2,31 @@
...
     {% unless page_title contains shop.name %}
      - {{ shop.name }}
     {% endunless %}
 </title>
 {% if page_description %}
     <meta name="description" content="{{ page_description | escape }}" />
 {% endif %}
+
+<script>
+    console.log("{{page_title}}");
```
Comment (ilyawzrd) [inline snippets/meta/meta-info.liquid:29]: Remove console logs

**Review by ilyawzrd — CHANGES_REQUESTED** [review]
remove console logs

File: snippets/meta/meta-info.liquid (hunk 1 above)
Comment (trueboroda) [reply snippets/meta/meta-info.liquid:29]: Done

Comment (pushnitsa) [reply snippets/meta/meta-info.liquid:29]: Forgot to push commit?

**Review by ilyawzrd — APPROVED** [review]

**Pull request merged**